eq-author-api/tests/utils/executeQuery.js

```diff
@@ -18,6 +18,18 @@ Resulted in:
 ${response.errors.map(e => e.message).join("\n----\n")}
     `);
   }
+  // Required because dynamoose transaction does not update the questionnaireModel
+  // originalItem is out of sync
+  if (
+    ctx.questionnaire &&
+    ctx.questionnaire.originalItem &&
```

Comment:
This questionnaire is the dynamoose model right? When would it not have an originalItem property?

Reply:
yes, good point. I'll remove this

Reply:
```suggestion
```

Reply:
Ah, I just checked and it has no original item when a questionnaire is duplicated